src/engraving/dom/box.cpp

```diff
@@ -655,7 +671,7 @@ void FBox::add(EngravingItem* e)
 TBox::TBox(System* parent)
     : VBox(ElementType::TBOX, parent)
 {
-    setBoxHeight(Spatium(1));
+    setBoxHeight(propertyDefault(Pid::BOX_HEIGHT).value<Spatium>());
```

Comment:
Might be nicer to use `resetProperty` here; I believe that's done in constructors of several other EngravingItem subclasses